controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -56,6 +57,7 @@ import (
 var (
 	nodeName                                string
 	nodeRunningUpdateRetryTime              = 5 * time.Second
+	defaultMaxNodeCapacity                  = 1
```

Comment:
1% is two nodes at a 200 cluster size, I think we have to use something else for really sequencial rollout.